dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS/Utilities.pm

```diff
@@ -327,6 +327,54 @@ sub map_type {
   return $type;
 }
 
+=head2 C<standard_XS_version_cmp()>
+
+=over 4
+
+=item * Purpose
+
+Provides the definition for PERL_VERSION_* compare macros.
+
+=item * Arguments
+
+None.
+
+=item * Return Value
+
+Returns a string to define PERL_VERSION.
+
+=back
+
+=cut
+
+sub standard_XS_version_cmp {
+    # these macros are now part of the API since 5.033
+    if ( $] > 5.033 ) {
+        # provided by handy.h / perl.h
+        return '';
+    }
+
+    # note ppport.h also provides these helpers
+    return <<"EOS";
+/* This stuff is not part of the API before 5.33.1 */
```

Comment:
Given the `#ifndef`s, I don't really see the harm of leaving this in here. That may be more robust anyway.

Reply:
The code as it (without any changes), is not incorrect as it s protected with an `#ifndef`.

But we currently do not need this code at all with `Perl >= 5.33.1`.
This is avoiding adding unnecessary macro definition / pollution to c files.
Also note that comment: `This stuff is not part of the API! You have been warned.` 
this is not true anymore with the last state of blead, all the most we will not use that code at all.

Reply:
This means that the output of EU::PXS will depend on the version of perl and not just the inputs and the version of EU::PXS which doesn't seem to be true currently.

Reply:
Indeed if we have case where we use two different versions of Perl this will be an issue... and leaving the code as it will avoid such failures.
I was not considering this case to be the `happy path` (I mean the most currently used path), but if indeed it exists in the wild ... you should then consider closing this case...